bin/varnishd/cache/cache_rfc2616.c

```diff
@@ -293,10 +293,6 @@ RFC2616_Do_Cond(const struct req *req)
 				return (0);
 			return (1);
 		}
-		AZ(ObjGetDouble(req->wrk, req->objcore, OA_LASTMODIFIED, &lm));
-		if (lm > ims)
-			return (0);
-		return (1);
 	}
```

Comment:
According to GCOV, we have test-cases which get here and both return()s are counted:

http://varnish-cache.org/gcov/bin/varnishd/cache/cache_rfc2616.c.html

Reply:
Yes, this code was hit by the vtc which I touched for this rfc:https://github.com/varnishcache/varnish-cache/pull/3118/files#diff-fd18bd93edbb1b820ca7213a950189a6